tests/collections/collection.test.ts

```diff
@@ -349,6 +400,20 @@ describe(`StargateMongoose - ${testClientName} Connection - collections.collecti
             assert.strictEqual(findResDocs.length, 1);
             assert.strictEqual(findResDocs[0]._id, idToCheck);
         });
+        it('should find & findOne L1 Boolean NE $ne document', async () => {
+            const doc1 = createSampleDocWithMultiLevel();
+            const doc2 = createSampleDoc2WithMultiLevel();
+            const insertDocResp1 = await collection.insertOne(doc1);
+            const insertDocResp2 = await collection.insertOne(doc2);
+            const idToCheck2 = insertDocResp2.insertedId;
+            const filter = { 'username': { '$ne': doc1.username } };
```

Comment:
This doesn't look like it is testing Boolean like the test title says. I think this test is identical to "should find & findOne L1 String NE $ne document"

Reply:
solved, thanks!